src/dyad/dtl/ucx_dtl.c

```diff
@@ -214,10 +219,10 @@ ucx_free_buf_done:;
     return rc;
 }
 
-static inline ucs_status_ptr_t ucx_send_no_wait (const dyad_ctx_t* ctx, void* buf, size_t buflen)
+static inline ucs_status_ptr_t ucx_send_no_wait (const dyad_ctx_t* ctx, bool is_warmup, void* buf, size_t buflen)
```

Comment:
Need a comment to explain the behavior with this argument.

Reply:
Added